### PR TITLE
Do not format generated files

### PR DESCRIPTION
**Pull request by Dschoordsch**
Commit pg:build files in unformatted form.
For VSCode users switch from builtin formatter to prettier extension
one. The builtin one formats different from prettier anyways.
Add the extension as recommendation.
There are two .prettierignore files added since VSCode extension looks
in the root directory while git commit hooks check the package.

### CODE REVIEW

**Comment by Dschoordsch** [comment]
We should do this or #5280

**Comment by tiffanyhan** [comment]
i pulled the remote branch to check it out. it looks like upon saving the generated files, the code still gets formatted, even though `.prettierignore` is present. not sure why `.prettierignore` is getting ignored (no pun intended)

**Comment by Dschoordsch** [comment]
The file should really be in the root directory next to the `.prettierrc` but then it is ignored during pre-commit formatting. Maybe just putting 2 of those there would solve it, I will check.

**Comment by Dschoordsch** [comment]
@tiffanyhan Could you check again if this works for you now?

**Comment by tiffanyhan** [comment]
it worked for me! this is so great, looks like the extra setting instructions to vscode made the difference.

**Review by tiffanyhan — APPROVED** [review]

**Comment by Dschoordsch** [comment]
@mattkrick is this ok to merge?

File: .vscode/settings.json [hunk 1]
```diff
@@ -26,6 +26,10 @@
 		}
 	],
 	"npm.packageManager": "yarn",
+	"editor.defaultFormatter": "esbenp.prettier-vscode",
```
Comment (mattkrick) [inline .vscode/settings.json:29]: out .eslint has prettier built in (https://github.com/ParabolInc/parabol/blob/1b2eb07d7804280847c616bdc372b7c61c83f0cb/.eslintrc.js#L3)
I wonder it hasn't been working :thinking:.
Happy to make the switch! At least on my machine there's some kind of memory leak with eslint that causes it to spin forever when trying to format. maybe only prettier will fix it!

**Pull request merged**